### ISSUE TITLE
Create test project for Events Functions 

### ISSUE THREAD

**Issue by acn-sbuad** [Altinn/altinn-events#188]
### Description

The events functions project does not have a unit test project. 
This has been postponed, but as the functions increase in number and complexity this should be set up. 
There might not be many relevant test cases at the moment, but for those that exists, they should be covered. 

e.g. verifying the client implementation and that accesstoken is set up for the request message.

Technical debt after #112 

### Tasks

- [ ] Set up project in SonarCloud
- [x] Create test project
- [ ] Implement relevant tests
- [x] Update analysis workflows to run tests and report results
- [ ] QA

### Acceptance Criterias

- Test project exists
- Test project is run for pull requests touching the code.
